specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -190,6 +190,11 @@ Provides settings for capturing photos with CameraCaptureUI.
 The settings include aspect ratio, image size, format, resolution, 
 and whether or not cropping is allowed by the user interface (UI).
 
+**Important Note**:  
+- Don't specify both size and aspect ratio in the `PhotoSettings`, doing so will cause an invalid argument exception.  
+- `PhotoSettings` can't have a ratio or size specified with cropping disabled.
```

Comment:
Should we specify on the behavior if ratio or size is specified when cropping is disabled? Whether API will throw exception?

Reply:
Yeah that makes sense, updated!